Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN addgroup -g 1000 -S las2peer && \
     adduser -u 1000 -S las2peer -G las2peer
 
 RUN mkdir -p /opt/feedback && chown -R 1000:1000 /opt/feedback && chmod -R "a+rwX" /opt/feedback
+RUN apk add dos2unix
+RUN find /<ordner> -type f -print0 | xargs -0 dos2unix
```

Comment:
This line was added to tackle formatting problems with line ending for Windows. With actual pipeline this should not be a problem